### ISSUE TITLE
mis-match between the paper and the repo.

### ISSUE THREAD

**Issue by MjdMahasneh** [ku-milab/MGICNN#9]
Hello, 

Thank you for sharing your work. I have noticed a mis-match between the paper and the repo:

1-in the paper, you mention using RELU, while in the code you're using LeakyRELU.
2-in the paper, you mention using SGD, while in the code you're using ADAM.
3-in the paper, you mention using a decaying learning rate (by a factor of 2.5% for each epoch), while in the code you're using a static learning rate.

Could you kindly elaborate on which settings to use?

Many thanks :)

**Comment by torki-hossein** [comment]
Hi, any reason for this Q?
https://github.com/ku-milab/MGICNN/issues/9#issue-956794811

**Comment by MjdMahasneh** [comment]
@torki-hossein Yes, I am actually trying to reproduce this work for comparative purposes.

**Comment by torki-hossein** [comment]
> @torki-hossein Yes, I am actually trying to reproduce this work for comparative purposes.

I'm not the author of this work, and I am also trying to reproduce this work for comparative and extending  purposes. please answer my mail (in your mailbox). ;)
